phply/phplex.py

```diff
@@ -515,8 +515,11 @@ def current_state(self):
     def input(self, input):
         self.lexer.input(input)
 
+    def next_lexer_token(self):
```

Comment:
Could you add a comment about what this is useful for? (both so others know and so it doesn't get removed as not useful in the future)

Reply:
Good point, docs added.